bin/mc-send-to-console

```diff
@@ -13,7 +13,11 @@ if [ ! -p "${CONSOLE_IN_NAMED_PIPE}" ]; then
 fi
 
 if [ "$(id -u)" = 0 ]; then
-  gosu minecraft bash -c "echo $* > '${CONSOLE_IN_NAMED_PIPE:-/tmp/minecraft-console-in}'"
+  if [[ getDistro == alpine ]]; then
```

Comment:
I think this will need to be the following to actually run the function and not be a bare string "getDistro", which will never equal "alpine":

```suggestion
  if [[ $(getDistro) == alpine ]]; then
```

Reply:
Yes, again I have tested with wrong command zsh suggested me and ran build without args :facepalm: Commit fixing it is there.

Reply:
Totally understand. I am grateful that IntelliJ makes it easy for me to switch between build types:

![image](https://user-images.githubusercontent.com/988985/163829688-fc59f38a-ba29-409b-a86a-8c933fc70cd7.png)